exercises/practice/darts/.docs/instructions.md

```diff
@@ -14,4 +14,6 @@ In our particular instance of the game, the target rewards with 4 different amou
 
 The outer circle has a radius of 10 units (This is equivalent to the total radius for the entire target), the middle circle a radius of 5 units, and the inner circle a radius of 1. Of course, they are all centered to the same point (That is, the circles are [concentric](http://mathworld.wolfram.com/ConcentricCircles.html)) defined by the coordinates (0, 0).
 
+To know whether a point is inside a circle (not _on_ the circumference), you can check whether `(x - center_x) ** 2 + (y - center_y) ** 2 < radius ** 2`, where `x` and `y` are the passed coordinates, `center_x` and `center_y` are the coordinates of the center of the circle, and `radius` is the radius of the circle.
```

Comment:
This may be written as a more standard formula.  This formula appears as though it may be Python code.  Not unhelpful, for sure, but if this goes to all the tracks that use this exercise, it might be something to consider.

Reply:
Good point!